tests/www/test_decorators.py

```diff
@@ -0,0 +1,28 @@
+from airflow.www import app
+from airflow.www.decorators import has_dag_access
+from flask_appbuilder import BaseView
+from unittest import TestCase
+from mock import Mock
+
+
+class TestDecorators(TestCase):
```

Comment:
```suggestion
class TestDecorators:
```

With pytest being used now we don't need to inherit from TestCase anymore